test_driver/mocks/services/mock_auth_service.dart

```diff
@@ -14,6 +14,9 @@ class MockAuthService implements AuthService {
   @override
```

Comment:
I think this should be FakeAuthService rather than MockAuthService - in Mockito terms I think this is a "real object" rather than a Fake but I think calling it a fake is the clearest documentation, IMO.